specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/readme.md

```diff
@@ -0,0 +1,71 @@
+# Cognitive Services Language SDK
```

Comment:
This needs to be merged with the readme.md file directly under the "Language" folder - not here. Basiclaly, you should change the `tag` near the top (along with the text about the current release) and add the section for the new version.

Reply:
fixed.